torchgeo/datasets/satlas.py

```diff
@@ -0,0 +1,700 @@
+    Sentinel-2 and NAIP, and can inform numerous applications from tackling illegal
+    deforestation to monitoring marine infrastructure. However, developing automatic
+    computer vision systems to parse these images requires a huge amount of manual
+    labeling of training data. By combining over 30 TB of satellite images with 137
+    label categories, SatlasPretrain serves as an effective pre-training dataset that
+    greatly reduces the effort needed to develop robust models for downstream satellite
+    image applications.
+
+    Reference implementation:
+
+    * https://github.com/allenai/satlas/blob/main/satlas/model/dataset.py
+
+    If you use this dataset in your research, please cite the following paper:
+
+    * https://doi.org/10.48550/arXiv.2211.15660
+
+    .. versionadded:: 0.6
+    """
+
+    # https://github.com/allenai/satlas/blob/main/satlaspretrain_urls.txt
+    urls: ClassVar[dict[str, tuple[str, ...]]] = {
+        'landsat': (
+            'https://pub-956f3eb0f5974f37b9228e0a62f449bf.r2.dev/satlaspretrain/satlas-dataset-v1-landsat.tar',
+        ),
+        'naip': (
+            'https://ai2-public-datasets.s3.amazonaws.com/satlas/satlas-dataset-v1-naip-2011.tar',
+            'https://ai2-public-datasets.s3.amazonaws.com/satlas/satlas-dataset-v1-naip-2012.tar',
+            'https://ai2-public-datasets.s3.amazonaws.com/satlas/satlas-dataset-v1-naip-2013.tar',
+            'https://ai2-public-datasets.s3.amazonaws.com/satlas/satlas-dataset-v1-naip-2014.tar',
+            'https://ai2-public-datasets.s3.amazonaws.com/satlas/satlas-dataset-v1-naip-2015.tar',
+            'https://ai2-public-datasets.s3.amazonaws.com/satlas/satlas-dataset-v1-naip-2016.tar',
+            'https://ai2-public-datasets.s3.amazonaws.com/satlas/satlas-dataset-v1-naip-2017.tar',
+            'https://ai2-public-datasets.s3.amazonaws.com/satlas/satlas-dataset-v1-naip-2018.tar',
+            'https://ai2-public-datasets.s3.amazonaws.com/satlas/satlas-dataset-v1-naip-2019.tar',
+            'https://ai2-public-datasets.s3.amazonaws.com/satlas/satlas-dataset-v1-naip-2020.tar',
+        ),
+        'sentinel1': (
+            'https://pub-956f3eb0f5974f37b9228e0a62f449bf.r2.dev/satlaspretrain/satlas-dataset-v1-sentinel1.tar',
+        ),
+        'sentinel2': (
+            'https://ai2-public-datasets.s3.amazonaws.com/satlas/satlas-dataset-v1-sentinel2-a.tar',
+            'https://ai2-public-datasets.s3.amazonaws.com/satlas/satlas-dataset-v1-sentinel2-b.tar',
+        ),
+        'labels': (
+            'https://ai2-public-datasets.s3.amazonaws.com/satlas/satlas-dataset-v1-labels-dynamic.tar',
+            'https://ai2-public-datasets.s3.amazonaws.com/satlas/satlas-dataset-v1-labels-static.tar',
+        ),
+        'metadata': (
+            'https://ai2-public-datasets.s3.amazonaws.com/satlas/satlas-dataset-v1-metadata.tar',
+        ),
+    }
+    # TODO
```

Comment:
Can anyone help me compute these MD5s? We're downloading all the non-NAIP data but likely won't be using NAIP in the near future.